invoicing/utils.py

```diff
@@ -19,7 +19,7 @@ def generate_reference_number(identifier: Union[int, str]) -> str:
     coefficients = (7, 3, 1)
     check_digit = (
         10 - sum(int(d) * c for d, c in zip(reversed(actual), cycle(coefficients))) % 10
-    )
+    ) % 10
```

Comment:
A test case for this would be good to have. Just call generate_reference_number with a number that would product invalid value with the unfixed code.

Reply:
Added test case